packages/compass-e2e-tests/.prettierignore

```diff
@@ -3,3 +3,7 @@
 fixtures
 .nyc_output
 coverage
+
+# Generated files
+electron-versions.json
+write-electron-versions.js
```

Comment:
These got created on my system while building the app and kept the `check` from passing locally.